test/libyul/evmCodeTransform/stackReuse/for_1.yul

```diff
@@ -2,18 +2,15 @@
 // ====
 // stackOptimization: true
 // ----
-// PUSH1 0x0
```

Comment:
Why is the format changed here?

Reply:
Is this test still valid at all? It was testing the old stack optimizer, right?

Reply:
It's code transform tests - I rather don't know why we assembled and disassembled stuff before yielding cryptic bytecodes instead of just actually testing what the code transform does, i.e. output evmasm.
And yes, this used to test the old stack optimizer - we could add a setting for still testing that instead, especially if we don't switch to the new code transform everywhere right away. But the idea was that it doesn't hurt to use the same tests for the new code transform.